core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1299,8 +1299,8 @@ public static boolean isClassProperty(String key) {
    */
   public static <T> T createTableInstanceFromPropertyName(AccumuloConfiguration conf,
       Property property, Class<T> base, T defaultInstance) {
-    String clazzName = conf.get(property);
-    String context = conf.get(TABLE_CLASSPATH);
+    String clazzName = conf.getWithoutWatch(property);
+    String context = conf.getWithoutWatch(TABLE_CLASSPATH);
```

Comment:
@ctubbsii is this what you are referring to for the per table context classpath?  This method is called in these places:

- PerTableVolumeChooser.java for TABLE_VOLUME_CHOOSER
- Tablet.java for      TABLE_COMPACTION_STRATEGY
- TabletServerResourceManager.java for     TABLE_COMPACTION_STRATEGY

Reply:
Yes, but that was just an example.